app/controllers/rankings_controller.rb

```diff
@@ -22,6 +24,28 @@ def valid_secret_id?
     params[:secret_id] == @participant.secret_id
   end
 
+  def create_rankings_hash
```

Comment:
Using a hash seems like a good constant time solution for this. Shows that software engineering interviews do have some sort of point in testing big O!